chain/client/src/stateless_validation/processing_tracker.rs

```diff
@@ -0,0 +1,112 @@
+use std::sync::Arc;
+
+use once_cell::sync::OnceCell;
+
+/// `ProcessingDoneTracker` can be used in conjuction with a `ProcessingDoneWaiter`
+/// to wait until some processing is finished. `ProcessingDoneTracker` should be
+/// kept alive as long as the processing is ongoing, then once it's dropped,
+/// the paired `ProcessingDoneWaiter` will be notified that the processing has finished.\
```

Comment:
```suggestion
/// the paired `ProcessingDoneWaiter` will be notified that the processing has finished.
```
Please remove slashes in the end here and below.
I personally limit all comment width by 80 symbols, however it wasn't strictly defined anywhere.

Reply:
The slashes translate to `<br>` in the documentation generated by `cargo doc`.
Without the slashes all of the information is in a single (wrapped) line, which is often less readable than separate lines.
I got into the habit of adding them in the previous project I worked on, but I can remove them, as cargo doc isn't really used here.

Reply:
Yeah, we don't use it here... I always read the code together with comments as a text and slashes would confuse me.
However, if it improves development experience, feel free to suggest it in Zulip Rust channel :)

Reply:
> However, if it improves development experience, feel free to suggest it in Zulip Rust channel :)

Eh it probably wouldn't. The previous project was a library crate, with its own `docs.rs` page, so it was important to have it look good. But this is a binary so things are different.